charts/simple-workflow/templates/workflow-pipeline.yaml

```diff
@@ -176,6 +193,16 @@ spec:
               {{- else }}
               value: status.sync.status == Synced, status.health.status == Healthy, status.sync.comparedTo.source.targetRevision == {{ $.Values.app.targetRevision }}
               {{- end }}
+       {{- range $jobName, $jobConfig := $config.jobs }}
+        - name: verify-job-synced-and-healthy-{{ $name }}
+          depends: verify-app-synced-and-healthy-{{ $name }}.Succeeded
```

Comment:
Why does it depend on app-synced-and-healthy ?

Reply:
duh, copy pasta err.

Reply:
If there's a logical dependency between app and the job, job config should be a sub dictionary of app config.

Reply:
it is actually. checkout https://github.com/Nextdoor/k8s-charts/pull/227/files#diff-15e2fdd4191db2b63c40fa4fb6dd7fab5efb400112e6b51f61fdfb66f2d6048c